include/ddc/detail/type_seq.hpp

```diff
@@ -161,6 +161,12 @@ struct ToTypeSeq;
 
 } // namespace detail
 
+template <class TypeSeq>
+constexpr std::size_t type_seq_size_v = std::numeric_limits<std::size_t>::max();
```

Comment:
Can we add a short docstring?

Reply:
This line is kind of a detail of implementation to me. I may change the way it is defined later to only keep the relevant part in the public namespace.

Reply:
I agree. I just thought that the docstring had to be on the most general case for it to appear correctly in the documentation

Reply:
Can be, I need to move `type_seq.hpp` out of the detail directory also to make it appear on the website.

Reply:
Ok. In that case it is not critical, especially for this case where the meaning is obvious, but I don't see any harm in adding docs now so they are ready if/when you expose type seq

Reply:
Sorry I thought it was on the `TypeSeq` specialization but I did not save the file.